projected_cloud_topic: "/cloud_projected"
image_topic: "/undistorted_image"